webapp/channels/src/components/drafts/panel/panel_header.test.tsx

```diff
@@ -10,11 +10,13 @@ import PanelHeader from './panel_header';
 
 describe('components/drafts/panel/panel_header', () => {
     const baseProps = {
```

Comment:
Nitpick: Unless we don't want to set all the props, we can avoid the `as const` and have this better type safe for future refactors by specifying the type here:

```suggestion
    const baseProps: React.ComponentProps<PanelHeader> = {
```

Reply:
Done.